scripts/obtain/dotnet-install.ps1

```diff
@@ -10,18 +10,14 @@
     Installs dotnet cli. If dotnet installation already exists in the given directory
     it will update it only if the requested version differs from the one already installed.
 .PARAMETER Channel
-    Default: preview
-    Channel is the way of reasoning about stability and quality of dotnet. This parameter takes one of the values:
-    - future - Possibly unstable, frequently changing, may contain new finished and unfinished features
-    - preview - Pre-release stable with known issues and feature gaps
-    - production - Most stable releases
+    Default: master
```

Comment:
I think there's a mis-match of what we want and what we have here.  I've set up a meeting to discuss tomorrow.

No need to block on this for now.  We can change it again if we decide on something different.